ts/screens/wallet/WalletHomeScreen.tsx

```diff
@@ -222,9 +218,7 @@ class WalletHomeScreen extends React.PureComponent<Props, State> {
 
     // To maintain retro compatibility, if the opt-in payment methods feature flag is turned off,
```

Comment:
Shouldn't this comment be removed/updated?

Reply:
```suggestion
```

sure